src/utils/nostr.ts

```diff
@@ -1,20 +1,20 @@
-import {
-  SimplePool,
-  generatePrivateKey,
-  getPublicKey,
-  getEventHash,
-  getSignature,
-  nip04,
-} from "nostr-tools";
+import {generatePrivateKey, getEventHash, getPublicKey, getSignature, nip04, SimplePool,} from "nostr-tools";
```

Comment:
should we follow a line length guideline?

Reply:
yes
we can use prettier for it.

I will raise another PR with prettier configured